tests/foreman/cli/test_ping.py

```diff
@@ -17,6 +17,7 @@
 pytestmark = [pytest.mark.tier1, pytest.mark.upgrade]
 
 
+@pytest.mark.pit_server
 @pytest.mark.parametrize('switch_user', [False, True], ids=['root', 'non-root'])
 def test_positive_ping(target_sat, switch_user):
```

Comment:
We also have the API ping test, Both of these tests could be good candidates for pit testing, right?

Reply:
@pondrejk ^

Reply:
hi, well, I'd say one is enough for this scenario, hammer goes through the api anyway, so if there's any problem there, we should see that with this test too